### PR TITLE
Refactor joystick commander example

### PR DESCRIPTION
**Pull request by jlamb-at-polysync**

### CODE REVIEW

File: joystick_commander/include/messages.h [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+
+
+#include "polysync_core.h"
+
+
+
+
+/**
+ * @brief Enable controls command identifier.
```
Comment (ltbj) [inline joystick_commander/include/messages.h]: *Disable

Comment (jlamb-at-polysync) [reply joystick_commander/include/messages.h]: nice, will fix

File: joystick_commander/src/node.c [hunk 2]
```diff
@@ -82,61 +72,14 @@ typedef struct
...
-
-
-/**
  * @brief Warning string.
  *
  */
 static const char WARNING_STRING[] =
 "\nWARNING: example is built for "
 "the Joystick device: Logitech Gamepad F310\n"
+"Back mode switch: 'X' setting\n"
```
Comment (ltbj) [inline joystick_commander/src/node.c:81]: What  is a back mode switch? 

'X' button, not switch?

**Comment by ltbj** [comment]
Can you please point this to branch "release-2.0.8-pr.8"?

We will merge that branch into master on release day.

File: joystick_commander/src/node.c (hunk 2 above)
Comment (jlamb-at-polysync) [reply joystick_commander/src/node.c:81]: there is a switch on the back of the controller, it has two positions one being labelled 'X'

**Pull request merged**